<launch>
  <!-- Set path to the configuration file -->
  <arg name="config_file" default="$(find rb_project)/config/simulator_config.yaml"/>

  <!-- Launch the simulator node -->
  <node name="simulator" pkg="rb_project" type="simulator" output="screen"
        args="$(arg config_file)">
  </node>

  <!-- Launch RViz with the specified configuration -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find rb_project)/config/robot_sim.rviz"/>
</launch>